localization/src/main/kotlin/edu/artic/localization/LocalizerModule.kt

```diff
@@ -0,0 +1,23 @@
+package edu.artic.localization
+
+import android.content.Context
+import dagger.Module
+import dagger.Provides
+import javax.inject.Singleton
+
+@Module
+abstract class LocalizerModule {
```

Comment:
rename to `LocalizationModule` to match the name of the module for discoverability.

Reply:
Sure.